src/pages/components/select/usage.mdx

```diff
@@ -158,7 +158,7 @@ the user is filling out forms. For full guidelines, refer to the
 <Row>
 <Column colLg={8}>
 
-![Example of a small select](images/select-usage-1.png)
+![Example of a small select](images/select-usage-1.gif)
```

Comment:
```suggestion
![Select invalid state](images/select-usage-1.gif)
```
isn't this GIF showing an invalid state rather than a small select?

Reply:
Good eye! Thanks!